src/pages/experiments/[experimentId]/data-processing/index.jsx

```diff
@@ -695,10 +695,15 @@ const DataProcessingPage = ({ experimentId, experimentData }) => {
                   Your navigation within Cellenics will be restricted during this time.
                   Do you want to start?
                 </p>
-                <Alert
-                  message='Note that you will lose all of your annotated cell sets.'
-                  type='warning'
-                />
+                {
+                  !(changedQCFilters.size === 1 && changedQCFilters.has('embeddingSettings'))
+                && (
+                  <Alert
+                    message='Note that you will lose your previous Louvain or Leiden clusters.'
```

Comment:
I did double check this specifically for custom cell sets and verified that we weren't losing them. Haven't checked for automatically annotated cell sets yet.

Reply:
The issue came up when trying to merge this to our fork. Martin pointed out that we weren't actually losing all cell sets and he specifically asked for this change (also to fix the fact that running embedding **doesn't** remove annotations).

Reply:
Automatically annotated cellsets aren't lost either. Not when rerunning clustering nor when changing QC settings.

Reply:
Yes but this warning is for any changes to QC -- I agree we could differentiate but it would get complicated:

1. only embedding (UMAP vs TSNE) settings --> lose nothing
2. only changing clustering settings --> lose louvain/leiden clustering only
3. changing anything else in QC --> lose all annotations

Reply:
I tested changing other settings in QC, and you still don't lose annotations.

Reply:
I also thought we would lose them, but I believe this was changed at some point.

Reply:
Ahh nice! Could have swarn I checked that but happy to be wrong -- apologies for the misunderstanding :)